lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -523,6 +523,33 @@ html.video-fullscreen {
       }
     }
 
+    .vert-due-date {
```

Comment:
Ick. I guess no way to avoid it, but don't love that the styling for these pills is so hardcoded here vs elsewhere. Ah well.

Reply:
Sorry! 😬